src/Maestro/Maestro.DataProviders/MaestroBarClient.cs

```diff
@@ -164,7 +185,10 @@ private SubscriptionPolicy ToClientModelSubscriptionPolicy(Maestro.Data.Models.S
 
         public async Task<IEnumerable<Subscription>> GetSubscriptionsAsync(string sourceRepo = null, string targetRepo = null, int? channelId = null)
         {
-            IQueryable<Data.Models.Subscription> query = _context.Subscriptions.Include(s => s.Channel);
+            IQueryable<Data.Models.Subscription> query = _context.Subscriptions
+                .Include(s => s.Channel)
+                .Include(s => s.LastAppliedBuild)
+                .Include(s => s.LastAppliedBuild.DependentBuildIds);
```

Comment:
If last applied build is null...does this fail?

Reply:
It won't fail because this only changes how SQL is generated but I now noticed that `DependentBuildIds` is marked as `[NotMapped]` so we won't be able to include it anyway. That means I need to get this information in separate query.

Reply:
This code was changed and this line is no longer needed